Lab1/app/src/main/java/com/example/jgozal1/lab1/MainActivity.java

```diff
@@ -0,0 +1,107 @@
+package com.example.jgozal1.lab1;
+
+
+        import android.app.AlertDialog;
+        import android.app.ListActivity;
+        import android.content.ContentValues;
+        import android.content.DialogInterface;
+        import android.database.Cursor;
+        import android.database.sqlite.SQLiteDatabase;
+        import android.os.Bundle;
+        import android.util.Log;
+        import android.view.Menu;
+        import android.view.MenuItem;
+        import android.view.View;
+        import android.widget.*;
+        import com.example.jgozal1.lab1.db.ItemContract;
+        import com.example.jgozal1.lab1.db.ItemDBHelper;
+
+public class MainActivity extends ListActivity {
+    private ListAdapter listAdapter;
+    private ItemDBHelper helper;
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.main);
+        updateUI();
+    }
+
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu,menu);
+        return true;
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.action_add_item:
+                AlertDialog.Builder builder = new AlertDialog.Builder(this);
+                builder.setTitle("Add an item");
+                builder.setMessage("What would you like to buy?");
+                final EditText inputField = new EditText(this);
+                builder.setView(inputField);
+                builder.setPositiveButton("Add", new DialogInterface.OnClickListener() {
+                    @Override
+                    public void onClick(DialogInterface dialogInterface, int i) {
+                        String item = inputField.getText().toString();
+
+                        helper = new ItemDBHelper(MainActivity.this);
+                        SQLiteDatabase db = helper.getWritableDatabase();
+                        ContentValues values = new ContentValues();
+
+                        values.clear();
+                        values.put(ItemContract.Columns.ITEM,item);
+
+                        db.insertWithOnConflict(ItemContract.TABLE,null,values,SQLiteDatabase.CONFLICT_IGNORE);
```

Comment:
You should probably do this database stuff in a function in your helper class to clean up the MainActivity code